package.json

```diff
@@ -51,5 +51,8 @@
     "lodash": "^4.17.21",
     "minimatch": "^5.1.0",
     "table": "^6.8.0"
+  },
+  "peerDependencies": {
+    "@jest/types": "*"
```

Comment:
Leaving this wide open so it can float with whatever version of types they likely already have installed.